site/src/pages/SetupPage/SetupPageView.tsx

```diff
@@ -164,7 +167,17 @@ export const SetupPageView: FC<SetupPageViewProps> = ({
             label={Language.passwordLabel}
             type="password"
           />
-
+          <TextField
+            {...getFieldHelpers("name")}
+            onBlur={(e) => {
+              e.target.value = e.target.value.trim();
+              form.handleChange(e);
+            }}
```

Comment:
Why do we need this?

Reply:
I want to ensure that any leading/trailing whitespace is removed before submission.

Reply:
I feel this should be done in the BE before saving the data on DB - so it is consistent between clients. We don't do that in any other form in the UI so, IMO, we can remove this for now.

Reply:
> We don't do that in any other form in the UI

I saw that it was done `AccountForm` so followed suit here.
IMO we are better off trimming in both places.

> I feel this should be done in the BE before saving the data on DB

`httpapi.UserRealNameValid` should handle this, but added an explicit call to `NormalizeRealUsername` for consistency with OIDC logins.